test/built-ins/RegExp/prototype/source/value-slash.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2016 Aleksey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-get-regexp.prototype.source
+es6id: 21.2.5.10
+description: Return value can be used to create an equivalent RegExp
+info: |
+  [...]
+  5. Let src be R.[[OriginalSource]].
+  6. Let flags be R.[[OriginalFlags]].
+  7. Return EscapeRegExpPattern(src, flags).
+
+  21.2.3.2.4 Runtime Semantics: EscapeRegExpPattern
+
+  [...] The code points / or any LineTerminator occurring in the pattern
+  shall be escaped in S as necessary to ensure that the String value
+  formed by concatenating the Strings "/", S, "/", and F can be parsed
+  (in an appropriate lexical context) as a RegularExpressionLiteral that
+  behaves identically to the constructed regular expression.
+---*/
+
+var re = eval('/' + new RegExp('/').source + '/');
+
+assert(re.test('/'), 'input: "/"');
+assert(re.test('_/_'), 'input: "_/_"');
+assert(!re.test('\\'), 'input: "\\"');
```

Comment:
please correct me if misunderstood:

- we can just assert that `new RegExp('/').source` is `'\/'`

however, according to the [spec](http://www.ecma-international.org/ecma-262/#sec-escaperegexppattern):

> For example, if P is "/", then S could be "\/" or "\u002F", among other possibilities, but not "/"